__tests__/workers/notifications/commentMention.ts

```diff
@@ -71,4 +91,81 @@ describe('commentMention', () => {
 
     expect(result).toBeUndefined();
   });
+
+  it('should not send notification when the parent commenter is mentioned', async () => {
+    const payload: Data = {
+      commentMention: {
+        commentId: 'c3',
+        commentByUserId: '2',
+        mentionedUserId: '1',
+      },
+    };
+
+    const result = await invokeNotificationWorker(
+      worker,
+      payload as unknown as Record<string, unknown>,
+    );
+
+    expect(result).toBeUndefined();
+  });
+
+  it('should not send notification when the author is mentioned', async () => {
```

Comment:
we don't send when author is mentioned because he will receive it either was as new comment notification?

Reply:
yes correct